### DISCUSSION TITLE
10月份更新后就变免费版了，无法登录专业版账号

### DISCUSSION THREAD

**Discussion by xiaoxin1982** [getlantern/lantern-forum-cn#961] [General]
10月份更新后就变免费版了，无法登录专业版账号，请帮忙解决，谢谢！账号：21217694@qq.com，
付款截图 
![Uploading 微信图片_20241013143109.jpg…]()

**Comment by xiaoxin1982** [comment]
![payment](https://github.com/user-attachments/assets/1b0ed203-29ed-4fce-a33e-3c4fda103ba3)

**Reply by xiaoxin1982** [reply]
没人吗，收了费用，不服务了？？？

**Comment by MaxMKMax** [comment]
@xiaoxin1982 好的. 已经帮您提交问题了. 不好意思让您久等了.

**Reply by xiaoxin1982** [reply]
谢谢，用电脑版授权过，也显示成功，但是手机依然是免费版，最近几天的上网效果很不好

**Reply by MaxMKMax** [reply]
@xiaoxin1982 您好. 让我再帮您问问看情况.

**Reply by xiaoxin1982** [reply]
还请帮忙加速一下啊，最近几天基本上和国外失去联系，蓝灯还是没法使用

**Reply by greySkiesMarsh** [reply]
@xiaoxin1982 Hello, I can see that your account is currently Pro. If you're having trouble getting logged into a mobile device, please follow the following instructions:

Android: To connect an Android device to your Lantern Pro account, first find the unique 6-digit device Linking code on your mobile device, found in Settings. Next, navigate to the “Authorize Device for Pro” window on your desktop device and hit the “Device Linking” button. Once you enter your device ID here, your Android device will be connected to your Pro account.

iOS: To connect an iOS device to your Lantern Pro account, first find the unique 6-digit device ID on your iOS device, found in Settings. Next, navigate to the “Authorize Device for Pro” window on your desktop device and hit the “Device Linking” button. Once you enter your device ID here, your iOS device will be connected to your Pro account.

**Reply by xiaoxin1982** [reply]
已经解决，是在22号左右，通过一次更新，随后我又通过window版本授权给手机，才变成专业版了，谢谢你们，但是发现最近这个软件比较不稳定，网速也是不如从前了，不知道为什么。

**Comment by ZHANGJIAMING19911991** [comment]
我也一样了，10月份之后就从专业版变成免费版了。2024年3月购买的，账号 363847091@qq.com。请问有大哥帮忙解决一下么？

**Reply by MaxMKMax** [reply]
@ZHANGJIAMING19911991 您好. 请您提供我支付单据. 我再帮您提交问题. 谢谢啦!

**Reply by ZHANGJIAMING19911991** [reply]
> @ZHANGJIAMING19911991 您好. 请您提供我支付单据. 我再帮您提交问题. 谢谢啦!

支付单据是支付宝，邮件里面的购买网站打不开了
![Uploading 微信图片_20241022220631.jpg…]()

**Reply by ZHANGJIAMING19911991** [reply]
![f0b10645a244621a3fbac3a8ae494c2](https://github.com/user-attachments/assets/6f8d71c7-86b8-40ee-9c74-6ac293b0a9a2)

**Reply by ZHANGJIAMING19911991** [reply]
![微信图片_20241022220631](https://github.com/user-attachments/assets/d314fcf9-8067-44fc-a551-adfa93ad8f60)

**Reply by greySkiesMarsh** [reply]
@ZHANGJIAMING19911991 Hi, please use this code to reactivate your account. PXX64-6B7M3-RD87C-M9TKK-YTJTG

**Reply by ZHANGJIAMING19911991** [reply]
> @ZHANGJIAMING19911991 Hi, please use this code to reactivate your account. PXX64-6B7M3-RD87C-M9TKK-YTJTG

Thank you, it works. But this code is only 1 year pro version? Shouldn't be 2 year plus  3 months bonus? (I bought from gogosn agent on 2024/03/24,it's 2 year+3months plan)

**Reply by ZHANGJIAMING19911991** [reply]
@greySkiesMarsh

**Reply by greySkiesMarsh** [reply]
@ZHANGJIAMING19911991 My apologies, I sent the code that most closely corresponded to the price that I could see in the screenshot you posted. I've updated the expiration date to 06/24/2026.

**Reply by ZHANGJIAMING19911991** [reply]
It's done, thx. @greySkiesMarsh
